src/constants/breakpoints.ts

```diff
@@ -1,7 +1,8 @@
 export const breakpoints = {
-  desktop: 1280,
-  phone: 600,
-  tablet: 720,
+  largeDesktop: 1200,
```

Comment:
where did you get 1200 from? Previously we were using 1280 and 1300 for the grid. I think we should be closer to that to avoid a difference in styling on the larger desktops (as much as possible)

Reply:
I got it from here: https://jira.zopa/browse/ZDS-19

Reply:
should we confirm breakpoints again with designers?